ani-cli

```diff
@@ -313,6 +317,7 @@ esac
 
 use_external_menu="${ANI_CLI_EXTERNAL_MENU:-0}"
 skip_intro="${ANI_CLI_SKIP_INTRO:-0}"
+skip_title="$ANI_CLI_SKIP_TITLE"
```

Comment:
Shellcheck complains but upon careful consideration, I find your current design more appealing than the workaround that would be required otherwise.

```suggestion
# shellcheck disable=SC2154
skip_title="$ANI_CLI_SKIP_TITLE"
```